### PR TITLE
add exit android app

### PR DESCRIPTION
**Pull request by festinalente91**
- now, user can exit pingo app if they press back button at the map scene.

### CODE REVIEW

**Comment by ochanje210** [comment]
should we have one more step before exiting the app, for example, showing alert whether user really want to exit the app?

**Comment by festinalente91** [comment]
I don't think so. Android users are already used to it.

**Comment by ochanje210** [comment]
this is part of design. should discuss with our designers

**Comment by festinalente91** [comment]
That would be even better :)

File: shared/containers/allLayout.js [hunk 1]
```diff
@@ -65,6 +65,13 @@ const styles = StyleSheet.create({
   }
 });
 
+let prevAndroidExitBtnClickTime = new Date().getTime();
+const ANDROID_EXIT = {
+  CLICK_TERM: 2000,
```
Comment (festinalente91) [inline shared/containers/allLayout.js]: doesn't matter, but that seems better :) I'll change that.

Comment (ochanje210) [inline shared/containers/allLayout.js]: thanks

**Comment by ochanje210** [comment]
shipit!

**Pull request merged**